docs/sphinx/features/device_allocators.rst

```diff
@@ -49,15 +50,11 @@ With the :class:`umpire::get_device_allocator` function, there is no need to kee
 call stacks can become quite complex. Users can instead use this function to obtain it inside whichever host or device
 function they need.
 
-Under the hood, the :class:`umpire::get_device_allocator` uses global arrays which can be accessed by both 
-the host and device. The global array is indexed by DeviceAllocator ID, which is returned by :class:`DeviceAllocator::getID()`. 
-Because we are using global arrays on host and device, the arrays need to be "set up" after at least one DeviceAllocator
-has been created, but before any kernels which use a DeviceAllocator are called. This process is done by calling 
-the ``UMPIRE_SET_UP_DEVICE_ALLOCS()`` macro. This just ensures that the host and device global arrays are updated and 
-pointing at the same memory.
-
 .. note::
-   In order to use the full capabilities of the DeviceAllocator, relocatable device code must be enabled.
+   In order to use the full capabilities of the DeviceAllocator, relocatable device code (RDC) must be enabled. If RDC is
```

Comment:
I think this note can be simplified, to just something like - "When compiling without relocatable device code, the UMPIRE_SET_UP_DEVICE_ALLOCATORS macro must be called in every translation unit that will use the get_device_allocator function"